imports/voting/startup/client/index.js

```diff
@@ -1,12 +1,12 @@
 import Vue from 'vue';
 import VueRouter from 'vue-router';
 import Vuex from 'vuex';
+import VueClipboards from 'vue-clipboards';
 import { sync } from 'vuex-router-sync';
 import VueMeteorTracker from 'vue-meteor-tracker';
 import { injectSupply } from 'vue-supply';
 import VueAutosize from 'vue-autosize';
 import createPersistedState from 'vuex-persistedstate';
-
```

Comment:
Why have you removed this line? Try don't commit changes that is not related to topic of current commit (77835d8856498aa4ae076fcb716156bc3fd103ac).